JobRunner/SpecialRunner.py

```diff
@@ -19,8 +19,8 @@ def __init__(self, config, job_id, logger=None):
         self.config = config
         self.top_job_id = job_id
         self.logger = logger
-        self.token = config["token"]
-        self.workdir = config.get("workdir", "/mnt/awe/condor")
+        self.token = config.token
+        self.workdir = config.workdir
```

Comment:
seems like, previously, default workdir for special runner is `/mnt/awe/condor` but now it's `/tmp/`. That will be fine right?

Reply:
We have definitely moved away from /mnt/awe, but there may be a preferred default other than /tmp.  I think the assumption is this would always get set.